flowey/flowey_core/src/node.rs

```diff
@@ -685,10 +706,10 @@ pub trait NodeCtxBackend {
         uses: &str,
         with: BTreeMap<String, ClaimedGhParam>,
         condvar: Option<String>,
-        outputs: BTreeMap<String, Vec<(String, bool)>>,
+        outputs: BTreeMap<String, Vec<(String, bool, bool)>>,
```

Comment:
if we do indeed need this extra bool added to these tuples... I think we should use this opportunity to introduce a proper set of `struct` types to use instead of these tuples, with field names that make the intent of each field clear

Reply:
You did the struct for `gh_to_rust` but not for `outputs`.